app/src/marketplace/utils/web3.js

```diff
@@ -58,32 +63,78 @@ export const getBalances = (): Promise<[BN, BN, BN]> => {
 }
 
 /**
- * Returns the DAI and ETH value equivalents for a given quanity of DATA through a 'Uniswap'.
- * A safety margin is added to ensure transactions succeed incase of slippage and token volatility.
+ * uniswapDATAtoETH & uniswapDATAtoDAI the DAI and ETH value equivalents for a given quanity of DATA through a 'Uniswap'.
```

Comment:
"uniswapDATAtoETH & uniswapDATAtoDAI ***return*** the DAI and ETH value equivalents for a given ***quantity*** of DATA through a 'Uniswap'"